daemon/graphdriver/overlay2/overlay.go

```diff
@@ -343,20 +363,23 @@ func (d *Driver) Create(id, parent string, opts *graphdriver.CreateOpts) (retErr
 		}
 	}()
 
+	driver := &Driver{}
```

Comment:
All the changes from here on seem no longer needed. and comments are probably out dated as well.